draft-ietf-quic-transport.md

```diff
@@ -3019,18 +3019,18 @@ frame is received:
 
 * The increase in ECT(0) and ECT(1) counters MUST be at least the number of QUIC
   packets newly acknowledged that were sent with the corresponding codepoint
-  minus the increase in the CE counter. Detects network remarking
+  minus the increase in the CE counter. This detects network remarking
   between ECT(0) and ECT(1).
 
 * The total increase in ECT(0), ECT(1), and CE counters reported in the ACK
   frame MUST be at least the total number of QUIC packets newly acknowledged in
-  this ACK frame. Detects if the network remark ECT(0), ECT(1) or CE to
+  this ACK frame. This detects if the network remark ECT(0), ECT(1) or CE to
   Not-ECT.
 
 If a sender receives an ACK that contains no new acknowledgments, for example
 due to reordering of the ACKs, then ECN counter comparison SHOULD NOT be
-performed. Also if sender do not have state to determine if a particular PSN
-is newly acknowledge or not, then the comparison SHOULD NOT be performed.
+performed. If the sender does not have state to determine if a particular PSN
+is newly acknowledged or not, then the comparison SHOULD NOT be performed.
 
```

Comment:
Please explain PSN - is it Packet Serial Number? In that case "packet", "packet number", or "PN" is more aligned with the rest of the text, I suppose.

Reply:
I think it's supposed to be packet sequence number?  But the text always uses packet number and abbreviates it to PN, so if that's what it is, it should be changed.

Reply:
Sorry, yes I will try to use PN in my comments.

Reply:
This uses packet number now.